charts/kubernetes-graphql-gateway/values.yaml

```diff
@@ -122,3 +122,19 @@ health:
     path: "/readyz"
     failureThreshold: 30
     periodSeconds: 10
+
+# -- Gateway API configuration
```

Comment:
for the documentation it is better to leave such comments on the individual items. If you leave it on the parent node it will print a rather unhelpful `object` entry with all children